vcd/resource_vcd_independent_disk.go

```diff
@@ -15,12 +15,15 @@ func resourceVcdIndependentDisk() *schema.Resource {
 		Create: resourceVcdIndependentDiskCreate,
 		Read:   resourceVcdIndependentDiskRead,
 		Delete: resourceVcdIndependentDiskDelete,
-
+		Importer: &schema.ResourceImporter{
+			State: resourceVcdIndependentDiskImport,
+		},
 		Schema: map[string]*schema.Schema{
 			"org": {
-				Type:     schema.TypeString,
-				Optional: true,
-				ForceNew: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				ForceNew:   true,
+				Deprecated: "Not needed anymore for this resource",
```

Comment:
How come org is "Not needed anymore"? How can you find the Org VDC if there are hundreds of Orgs? This is a big design change, as we have an `org` in all resources.

Reply:
Because we don't use or need it. Disk depends to vdc, not Org. https://code.vmware.com/apis/287/vcloud/doc/doc/operations/POST-CreateDisk.html. There isn't back compatibility issue as we didn't use `org` in previous code.

Reply:
We get the VDC using `_, vdc, err := vcdClient.GetOrgAndVdcFromResource(d)`
This function cannot return a VDC without Org. We choose to discard the Org from the results, but we can't get to the VDC without it.

Reply:
org deprecation removed